File: 2024-Semester-1/PB (VP)/Lab Exercises/Exercise Week 4 - 15.10.2024/ex-2.py
n = int(input("Enter the number of lines to print: "))

for i in range(n):
    print("*" * (i + 1))

# for i in range(n):
#     print("*" * (i + 1) + " " * (n - i - 1) + " " * (n - i - 1) + "*" * (i + 1))

# for i in range(n):
#     print(" " * (n - i - 1) + "*" * (2 * i + 1))

File: 2025-Semester-2/BPL (BPE)/Seminar Exercises/Exercise Week 7 - 19.03.2025/ex-2.c
//
// Created by Kristiyan Petsanov on 19.03.25.
//

#include <stdio.h>

int checkRelations(int arr[], int n) {
  for (int i = 1; i < n; i++) {
    if (i % 2 == 1) {
      if (arr[i] <= arr[i - 1]) {
        return 0;
      }
    } else {
      if (arr[i] >= arr[i - 1]) {
        return 0;
      }
    }
  }
  return 1;
}

int main() {
  int n;
  printf("Enter the number of elements in the array: ");
  scanf("%d", &n);

  int arr[n];
  printf("Enter the elements of the array: ");
  for (int i = 0; i < n; i++) {
    scanf("%d", &arr[i]);
  }

  if (checkRelations(arr, n)) {
    printf("The relations are satisfied.\n");
  } else {
    printf("The relations are not satisfied.\n");
  }

  return 0;
}